src/Core/Console/AutomaticInstallation.php

```diff
@@ -177,35 +182,34 @@ protected function doExecute()
 	}
 
 	/**
-	 * @param App $app
-	 * @return array
+	 * @param Install $install the Installer instance
 	 */
-	private function runBasicChecks($app)
+	private function runBasicChecks($install)
```

Comment:
Please add the `Install` type-hinting in the parameter list.

Reply:
Is this best practice everywhere or just in this case? Because I wasn't sure if it's PHP 5.x compatible.